packages/xo-web/src/xo-app/backup/new/_schedules/index.js

```diff
@@ -199,6 +199,7 @@ const Schedules = decorate([
           data-toggleScheduleState={effects.toggleScheduleState}
           individualActions={INDIVIDUAL_ACTIONS}
           rowTransform={state.rowTransform}
+          stateUrlParam='s'
```

Comment:
```suggestion
          stateUrlParam='s_schedules'
```

Reply:
Isn't it the main (obvious) table for this page?